packages/react-ui-core/src/Carousel/PhotoCarousel.js

```diff
@@ -29,12 +30,15 @@ export default class PhotoCarousel extends PureComponent {
 
   static defaultProps = {
     lazyLoad: false,
+    isBackgroundImage: false,
     theme: {},
   }
 
-  photo(path) {
+  photo(item) {
     const { server, dimensions } = this.props
-    return `${server}${path}${dimensions}`
+    const { url, path, id } = item
+
+    return url || `${server}${path || id}${dimensions}`
```

Comment:
why support path and id?

Reply:
Well, AG is currently using `id`.  I'm not opposed to removing it, but I'm assuming your path is something like `imgr/12345` or no?  If not then I guess they're synonymous?  If it's using `imgr` we'd either have to convert the bff or map over every photo to change the key name.